launcher/src/CmderLauncher.cpp

```diff
@@ -299,6 +316,116 @@ void UnregisterShellMenu(std::wstring opt, wchar_t* keyBaseName)
 	RegCloseKey(root);
 }
 
+struct cmderOptions
+{
+	std::wstring cmderCfgRoot = L"";
+	std::wstring cmderStart = L"";
+	std::wstring cmderTask = L"";
+	std::wstring cmderRegScope = L"USER";
+	bool cmderSingle = false;
+	bool registerApp = false;
+	bool unRegisterApp = false;
+	bool error = false;
+
+};
+
+cmderOptions GetOption()
+{
+	cmderOptions cmderOptions;
+	LPWSTR *szArgList;
+	int argCount;
+
+	szArgList = CommandLineToArgvW(GetCommandLine(), &argCount);
+	/*
```

Comment:
Any reason why these lines are left in commented out?

Reply:
Fixed!